src/test/test_utils.cpp

```diff
@@ -84,3 +84,19 @@ std::string readableError(const float* expected_output, const float* actual_outp
     }
     return ss.str();
 }
+
+std::function<bool(const tensorflow::TensorShapeProto&, const std::vector<int64_t>&&)> isShapeTheSame =
```

Comment:
Wouldnt it be better to create function instead of lambda? The point of having lambda in first place was not to move things to separate header file. Since we already move it now, what is the value of having lambda instead of regular function?